packages/gitbook/src/components/SpaceLayout/SpaceLayout.tsx

```diff
@@ -94,84 +94,82 @@ export async function SpaceLayout(props: {
-                            )
-                        }
-                        innerHeader={
-                            // displays the search button and/or the space dropdown in the ToC according to the header/variant settings. E.g if there is no header, the search button will be displayed in the ToC.
-                            <>
-                                {!withTopHeader && (
-                                    <div className={tcls('hidden', 'lg:block')}>
-                                        <React.Suspense fallback={null}>
-                                            <SearchButton>
-                                                <span className={tcls('flex-1')}>
-                                                    {t(
-                                                        getSpaceLanguage(customization),
-                                                        customization.aiSearch.enabled
-                                                            ? 'search_or_ask'
-                                                            : 'search',
-                                                    )}
-                                                </span>
-                                            </SearchButton>
-                                        </React.Suspense>
-                                    </div>
-                                )}
-                                {!withTopHeader && withSections && sections && (
-                                    <SiteSectionList
-                                        className={tcls('hidden', 'lg:block')}
-                                        sections={sections}
-                                    />
-                                )}
-                                {withVariants && (
-                                    <SpacesDropdown
-                                        space={space}
-                                        spaces={spaces}
-                                        className={tcls('w-full')}
-                                    />
+                    // Ensure the footer is display below the viewport even if the content is not enough
+                    `min-h-[calc(100vh-64px)]`,
+                    withTopHeader ? null : 'lg:min-h-screen',
+                )}
+            >
+                <TableOfContents
+                    space={space}
+                    customization={customization}
+                    content={content}
+                    pages={pages}
+                    ancestors={ancestors}
+                    context={contentRefContext}
+                    header={
+                        withTopHeader ? null : (
+                            <div
+                                className={tcls(
+                                    'hidden',
+                                    'pr-4',
+                                    'lg:flex',
+                                    'flex-grow-0',
```

Comment:
Unless the api changed and I missed it `flex-grow-0` is not the correct tailwind util, should be `grow-0`:
```suggestion
                                    'grow-0',
```